### ISSUE TITLE
Navigation Line

### ISSUE THREAD

**Issue by MrAdi149** [FireDragonGameStudio/ARIndoorNavigation-Unity6#1]
I just downloaded your repo, but after running it, navigation line is invisible

**Comment by FireDragonGameStudio** [comment]
Do you see the line in the editor? It's possible, that you'll have to bake the navigation mesh again.

**Comment by MrAdi149** [comment]
No, I am not able to see the line in the editor. I baked the navigation mesh again then also it's not working.

**Comment by FireDragonGameStudio** [comment]
Try to remove all walls, bake the NavMesh and try it agin. If no path can be calculated, you'll have to check if your general navmesh setup is correct. Without the walls, there should be at least a straight connection from start to target.

**Comment by MrAdi149** [comment]
Thanks for your response! I watched your AR indoor navigation video of your room and tried to create a similar app for my office, but the line rendering doesn't seem to work. I've uploaded some pictures to this link: [Google Drive](https://drive.google.com/drive/folders/1dWdhDcP-MrDrj-eWZLJZv8KQNRJPM9EZ?usp=sharing). Could you please check and let me know what might be causing the app not to work?


and for this project i tried to remove the walls and baked again then also it is not working

**Comment by e8-ChenDongLi** [comment]
@MrAdi149 Hello, I have encountered the same problem After installation on the phone, the navigation route does not display
Can you tell me how you solved it? Thank you.
